hydroflow_datalog_core/src/grammar.rs

```diff
@@ -179,16 +179,17 @@ pub mod datalog {
         Gt(#[rust_sitter::leaf(text = ">")] ()),
         GtEq(#[rust_sitter::leaf(text = ">=")] ()),
         Eq(#[rust_sitter::leaf(text = "==")] ()),
+        Neq(#[rust_sitter::leaf(text = "!=")] ()),
     }
 
     #[derive(Debug, Clone)]
     pub struct PredicateExpr {
         #[rust_sitter::leaf(text = "(")]
         _l_brace: (),
 
-        pub left: Spanned<Ident>,
+        pub left: Spanned<ValueExpr>,
```

Comment:
How do you feel about renaming ValueExpr to IntExpr?

Reply:
I like this idea